docs/installationguide/redhat.rst

```diff
@@ -53,7 +53,9 @@ ZoneMinder releases are now being hosted at RPM Fusion. New users should navigat
 
 Note that RHEL/CentOS 7 users should use yum instead of dnf.
 
-Once ZoneMinder has been installed, it is critically important that you read the README file under /usr/share/doc/zoneminder. ZoneMinder will not run without completing the steps outlined in the README.
+Once ZoneMinder has been installed, it is critically important that you read the README.Redhat7 file under /usr/share/doc/zoneminder-[version #] folder. ZoneMinder will not run without completing the steps outlined in the README.Redhat7 file.
+
+if you wish to get rig of a warning from your browser regarding certificate, you can follow suggestions in README.https file.
```

Comment:
I get it you are trying to be helpful, but this is already stated in step 6 of the README. Let's keep this note in just one place. If you want to modify step 6 of both README's to make that more clear, I'd be in favor of that.
The README's live here: https://github.com/ZoneMinder/zoneminder/tree/master/distros/redhat/readme

Reply:
Thank you. I agree with you. I missed it.